.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,3 +20,12 @@ Fixes #0000 <!-- link to issue if one exists -->
   Summary of the changes committed.
   Before / after screenshots appreciated for UI changes.
 -->
+
+### Checklist
+
+**Note**: once this PR is merged, it becomes a new release for this template.
+
+- [ ] I have added/updated tests for this change
+- [ ] I have incremented the version in the top-level `package.json` file **AND** in the `web/version.txt` file
```

Comment:
Do we still need to follow the top-level version?

Reply:
I think it's probably good practice to include it in the package.json too, no?